### PR TITLE
EAT-513 update hello world python

### PR DESCRIPTION
**Pull request by gregory**
- When the Partner requires new scopes, the merchant will go through a scope update flow
which will generate a new token for the new granted scope set.
The Partner should expect to receive a new access_token on the callback endpoint.
- Update the hello world app to highlight the scopes associated with the received token

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -144,14 +143,12 @@ def auth_callback():
...
         user.admin = True
-
     db.session.add(user)
     db.session.commit()
 
     # Log user in and redirect to app home
     flask.session['userid'] = user.id
-    return flask.redirect(flask.url_for('index'))
-
+    return flask.redirect(app.config['APP_URL'])
```
Comment (bc-croh92) [inline app.py]: Why did you change this?

**Comment by bc-croh92** [comment]
:+1:

File: app.py (hunk 1 above)
Comment (gregory) [inline app.py]: This is the bug we fixed together, as the route for index doesn't exists. This might not be the best fix, but it fixes it :)

**Comment by mattolson** [comment]
:+1:
